client/components/message-area.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+export default class MessageArea extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      roomId: this.props.roomId,
+      roomName: '',
+      roomMessages: []
+    };
+    this.adjustTextareaHeight = this.adjustTextareaHeight.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/rooms/${this.props.roomId}`)
+      .then(response => response.json())
+      .then(result => {
+        this.setState({
+          roomId: result.chatId,
+          roomName: result.name,
+          message: []
+        });
+      });
+  }
+
+  adjustTextareaHeight(event) {
+    event.target.style.height = '24px';
+    event.target.style.height = `${event.target.scrollHeight}px`;
+    event.target.parentElement.style.height = '36px';
+    event.target.parentElement.style.height = `${event.target.scrollHeight + 8}px`;
```

Comment:
We don't modify the DOM directly in React. Instead use state changes and pass props to influence your elements